cmd/database-recorder/database-recorder.go

```diff
@@ -35,7 +35,6 @@ func storeData(payload []byte) {
 	err := json.Unmarshal(payload, &sensorDataType)
 
 	doc := bson.M{
-		"sensorId":   sensorDataType.SensorId,
```

Comment:
It turns out he asked for this fieldt. See the page 4 of the [assessment](https://www.laurent-guerin.fr/imt-atlantique). Sorry, I told you we could remove it. We need it back :(